core/claims/claimAuthorizeKSignBabyJub_test.go

```diff
@@ -19,28 +20,88 @@ func TestClaimAuthorizeKSignBabyJub(t *testing.T) {
 	}
 	// Add input data to the test vector
 	if generateTest {
-		testgen.SetTestValue("privateKey", "28156abe7fe2fd433dc9df969286b96666489bac508612d0e16593e944c4f69f")
+		testgen.SetTestValue("0_privateKey", "28156abe7fe2fd433dc9df969286b96666489bac508612d0e16593e944c4f69f")
 	}
 	// Create new claim
 	var k babyjub.PrivateKey
-	hexK := testgen.GetTestValue("privateKey").(string)
+	hexK := testgen.GetTestValue("0_privateKey").(string)
 	if _, err := hex.Decode(k[:], []byte(hexK)); err != nil {
 		panic(err)
 	}
 	pk := k.Public()
 	c0 := NewClaimAuthorizeKSignBabyJub(pk)
+	assert.True(t, merkletree.CheckEntryInField(*c0.Entry()))
 	c0.Version = 1
 	e := c0.Entry()
 	// Check claim against test vector
-	checkClaim(e, t)
+	testgen.CheckTestValue(t, "0_HIndex", e.HIndex().Hex())
+	testgen.CheckTestValue(t, "0_HValue", e.HValue().Hex())
+	testgen.CheckTestValue(t, "0_dataString", e.Data.String())
 	dataTestOutput(&e.Data)
 	c1 := NewClaimAuthorizeKSignBabyJubFromEntry(e)
 	c2, err := NewClaimFromEntry(e)
 	assert.Nil(t, err)
 	assert.Equal(t, c0, c1)
 	assert.Equal(t, c0, c2)
+	assert.True(t, merkletree.CheckEntryInField(*e))
 	// Stop test (write new test vector if needed)
 	if err := testgen.StopTest(); err != nil {
 		panic(fmt.Errorf("Error stopping test: %w", err))
 	}
 }
+func TestClaimAuthorizeKSignBabyJub1(t *testing.T) {
+	// If generateTest is true, the checked values will be used to generate a test vector
+	generateTest := false
+	// Init test
+	if err := testgen.InitTest("claimAuthorizeKSignBabyJub", generateTest); err != nil {
```

Comment:
"claimAuthorizeKSignBabyJub" vectorTest has already been used in https://github.com/iden3/go-iden3-core/pull/264/commits/0de37bd015ba1c181e5f3c9fee1007c97c67a821#diff-0473308ee2b6d9453c4aa6d815355514R18